packages/styled/types/tests.tsx

```diff
@@ -1,35 +1,42 @@
+import * as React from 'react'
 import styled, { CreateStyled } from '@emotion/styled'
 
-// $ExpectType CreateStyledComponentIntrinsic<"a", {}, any>
+// $ExpectType CreateStyledComponent<{ theme?: any; }, DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>, { theme: any; }>
 styled.a
-// $ExpectType CreateStyledComponentIntrinsic<"body", {}, any>
+// $ExpectType CreateStyledComponent<{ theme?: any; }, DetailedHTMLProps<HTMLAttributes<HTMLBodyElement>, HTMLBodyElement>, { theme: any; }>
 styled.body
-// $ExpectType CreateStyledComponentIntrinsic<"div", {}, any>
+// $ExpectType CreateStyledComponent<{ theme?: any; }, DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>, { theme: any; }>
 styled.div
-// $ExpectType CreateStyledComponentIntrinsic<"svg", {}, any>
+// $ExpectType CreateStyledComponent<{ theme?: any; }, SVGProps<SVGSVGElement>, { theme: any; }>
 styled.svg
 
 {
-  // $ExpectType CreateStyledComponentIntrinsic<"svg", { bar: string }, { themed: "black" }>
-  styled.div<{ bar: string }, { themed: 'black' }>`
+  // $ExpectType StyledComponent<{ theme?: any; } & { bar: string; }, DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement>>
+  styled.div<{ bar: string }>`
```

Comment:
the intention for this "test" was to check if the given Theme (when it was a separate parameter) is available in interpolations on props, as with new typings theme is "just a prop" I guess there is no need for that original intention to be anyhow preserved 

👍

Reply:
Removed the outer assertion, left the inner one.